docs/reference/all_strategies.rst

```diff
@@ -120,6 +123,6 @@ Here are the docstrings of all the strategies in the library.
 .. automodule:: axelrod.strategies.titfortat
    :members:
    :undoc-members:
-.. automodule:: axelrod.strategies.gradualkiller
```

Comment:
I didn't, if you look further up you will see that I moved it so that the list was in alphabetical order again.